### PR TITLE
Updating projects.

### PR DESCRIPTION
**Pull request by nchronas**
Updating projects: documentation, make sure best practices are used and use of Dockerfile.templates

### CODE REVIEW

**Comment by nchronas** [comment]
Should we update the resin site screenshots as well?

File: Dockerfile.template [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+FROM resin/%%RESIN_MACHINE_NAME%%-python
+
+MAINTAINER Shaun Mulligan <shaun@resin.io>
+
+#switch on systemd init system in container
+ENV INITSYSTEM on
+
+# pip install python deps from requirements.txt
+# For caching until requirements.txt changes
+ADD ./requirements.txt /requirements.txt
```
Comment (Page-) [inline Dockerfile.template]: Can you change this to `COPY` please

File: Dockerfile.template [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+# pip install python deps from requirements.txt
+# For caching until requirements.txt changes
+COPY ./requirements.txt /requirements.txt
+RUN pip install -r /requirements.txt
+
+COPY . /usr/src/app
+WORKDIR /usr/src/app
+
+#run main.py when the container starts
+CMD ["python","/usr/src/app/src/main.py"]
```
Comment (shaunmulligan) [inline Dockerfile.template]: probably dont need the absolute path here is you define WORKDIR

**Comment by shaunmulligan** [comment]
I would probably change the screen shot if possible, even if you use staging

**Pull request merged**